docs/model_version_policy.md

```diff
@@ -28,5 +28,7 @@ Examples:
 
 - When the model version is deleted from the file system, it will become unavailable on the server and it will release RAM allocation. Updates in the deployed model version files will not be detected and they will not trigger changes in serving.
 
+> NOTE: On Windows, models which are loadeded by OpenVINO cannot be removed from disk until the model is unloaded. This is because OpenVINO by default uses `mmap` to speed up model loading time. To disable this feature, set `--plugin_config "{\"ENABLE_MMAP\":\"NO\"}"`.  
```

Comment:
A note might useful that this is applicable only for IR models. To be confirmed if that is really the case.